features/step_definitions/authorization_requests_steps.rb

```diff
@@ -275,3 +275,7 @@
     expect(page).to have_css('.authorization-request-state', text: I18n.t("authorization_request.status.#{state}"), count:)
   end
 end
+
+Et(/je peux voir le bouton "Débuter mon habilitation pour Portail HubEE - Démarche CertDC" grisé et désactivé/) do
+  expect(page).to have_css('button#start_authorization_request_authorization_request_form_portail-hubee-demarche-certdc.fr-btn.fr-icon-save-line.fr-btn--icon-left[disabled]', text: 'Débuter mon habilitation pour Portail HubEE - Démarche CertDC')
```

Comment:
```suggestion
Et("Et peux voir le bouton {text} grisé et désactivé") do |text|
  expect(page).to have_css('button[disabled]', text: )
```

Pour plusieurs raison:

1. `/whatever/` est pour utiliser des regex, il n'y a aucun regex ici autant utiliser le format `string`
2. Si demain on change le texte on veut seulement modifier le test cucumber et non l'étape, qui ici peut clairement être générique.